src/components/FeatureCars/FeaturedCars.tsx

```diff
@@ -1,12 +1,14 @@
 import React, { useCallback, useEffect, useState } from "react";
-import { Pagination } from "react-bootstrap";
+import "./FeaturedCars.scss";
+import CarCard from "../CarCard/CarCard";
+import { Button, Pagination } from "react-bootstrap";
 import { ArrowRight } from "react-bootstrap-icons";
```

Comment:
no need for this now right?

Reply:
View All should also have the right designer icon from figma

Reply:
Made this change